app/validators/metadata_validator.py

```diff
@@ -32,7 +32,14 @@ def validate_mandatory(self):
         # ru_name required for template rendering in business, default and NI theme
         required_metadata_names = []
 
-        if self.theme_name in ["business", "default", "northernireland", "beis", "orr"]:
+        if self.theme_name in [
+            "business",
+            "default",
+            "northernireland",
+            "beis",
+            "nibeis",
```

Comment:
To be consistent with census one in runner, this should be like `beis-ni`. Use a hyphen to make it clear what it is. `beis-northern-ireland` would be better for readability.

Reply:
In that case would you also want `northernireland` renamed to `northern-ireland` or `NI` as well?

Reply:
No, only because they are live surveys using "northernireland", so you can't rename it without those changing. Ideally yes but we cant right now.